spec/board_spec.rb

```diff
@@ -100,5 +100,103 @@
       expect(@board.valid_placement?(@cruiser, ['A2', 'B2', 'C2'])).to be(false)
     end
   end
+  
+  describe '#render' do
+    it 'can render a board without boats' do
+      expect(@board.render).to eq("  1 2 3 4 \nA . . . . \nB . . . . \nC . . . . \nD . . . . \n")
+    end
+
+    it 'can render a boat' do
+      expect(@board.render).to eq("  1 2 3 4 \nA . . . . \nB . . . . \nC . . . . \nD . . . . \n")
+      @board.place(@cruiser, ['A1', 'A2', 'A3'])
+      cell_1 = @board.cells['A1']
```

Comment:
I cant help but think lines 112 - 117 can just be removed because we are never actually calling anything on them? We are just trying to show that #render can render a ship on the board after it is placed. Line 118 accomplishes that so I dont think we need to show that .empty? is false on each cell also.